### PR TITLE
Docker proof

### PR DESCRIPTION
**Pull request by Sammers21**
A proof that docker repo works. 

Related to #154.

### CODE REVIEW

**Comment by Sammers21** [comment]
@g4s8, can you review?

File: examples/docker/run.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+# Wait for container to be ready for new connections.
+sleep 5
+
+# Pull an image from docker hub.
+docker pull ubuntu
+
+# Push the pulled image to artipie
+docker image tag ubuntu localhost:8080/my-docker/myfirstimage
+docker push localhost:8080/my-docker/myfirstimage
```
Comment (g4s8) [inline examples/docker/run.sh:15]: @Sammers21 I think we need to verify `pull` as well. You can test it with `docker image rm localhost:8080/my-docker/myfirstimage` and `docker pull localhost:8080/my-docker/myfirstimage`

**Comment by 0crat** [comment]
Job was finished in 0 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job audit: `ARC` didn't merge or close PR, performer @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)[/z](https://www.0crat.com/u/carlosmiranda) didn't make CR comments

**Comment by 0crat** [comment]
The project doesn't have enough funds, can't make a payment

**Pull request merged**